Source/Core/Common/SettingsHandler.cpp

```diff
@@ -11,9 +11,10 @@
 #include <string>
 
 #ifdef _WIN32
+#include <windows.h> //NOLINT
+// The following Windows headers must be included AFTER windows.h.
```

Comment:
> Should I really specify that?
Yes. The point of comments is to specify what and why. 

Also, why aren't you using precompiled headers on Windows? This issue shouldn't happen if you're using them.